enterprise/tiny-comments.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: default
+title: Tiny Comments
+description: Tiny Comments provides the ability to add comments to the content and collaborate with other users for content editing.
+keywords: enterprise pricing video youtube vimeo mp3 mp4 mov movie clip film comment commenting mediaembed media
+---
+
+The [Tiny Comment]({{ site.baseurl }}/plugins/tiny-comment/) plugin provides the ability to add comments to the content and collaborate with other users for content editing within the TinyMCE editor.
+The primary value that TinyComments offer is a user interface to create, reply, and delete comments. The TinyComments plugin is built upon the the new [Annotations API]({{ site.baseurl }}/plugins/annotations/) and uses annotations to create comment threads.
+
+> Note: The TinyComments plugin in its current state does not store the comments anywhere. However, it gives the integrator callback APIs to create, search, and store comment threads.
+
+Example:
```

Comment:
Worth noting that this example will just store the comments in memory. If the user refreshes the page in the browser then the comments will disappear.

Also, this example won't run as is, it's not a complete HTML document and doesn't include the editor. Do we want examples that readers copy and paste and go?